### DISCUSSION TITLE
viewport compositing and render are significantly different

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#551] [Q&A]
![image](https://github.com/bnpr/Malt/assets/36077492/d9d8fcd1-abf1-4f6c-8596-ccc3652d1919)
[SS_LightningArc_P_01.zip](https://github.com/bnpr/Malt/files/15175978/SS_LightningArc_P_01.zip)

**Comment by HSDSZ** [comment]
seems not malt related. Eevee also shows this difference
https://blenderartists.org/t/viewport-composite-different-from-render/1487218

**Reply by pragma37** [reply]
Yes, the CPU and GPU compositors are still not fully compatible:
https://projects.blender.org/blender/blender/issues/118548
